packages/delegation-toolkit/src/caveatBuilder/scope/nativeTokenTransferScope.ts

```diff
@@ -0,0 +1,30 @@
+import { createCaveatBuilder } from '../coreCaveatBuilder';
+import type { CoreCaveatBuilder } from '../coreCaveatBuilder';
+import type { NativeTokenTransferAmountBuilderConfig } from '../nativeTokenTransferAmountBuilder';
+import type { DeleGatorEnvironment } from 'src/types';
+
+export type NativeTokenTransferScopeConfig = {
+  type: 'nativeToken-transfer';
```

Comment:
what do you think about using the primary caveat name exported from the caveat builder?

```
import type { nativeTokenTransferAmount } from '../nativeTokenTransferAmountBuilder';

export type NativeTokenTransferScopeConfig = {
  type: nativeTokenTransferAmount
};
```